src/porepy/grids/gmsh/mesh_2_grid.py

```diff
@@ -28,9 +28,9 @@ def create_2d_grids(pts, cells, **kwargs):
     # List of 2D grids, one for each surface
     g_2d = []
     is_embedded = kwargs.get("is_embedded", False)
+    phys_names = kwargs.get("phys_names", False)
+    cell_info = kwargs.get("cell_info", False)
     if is_embedded:
-        phys_names = kwargs.get("phys_names", False)
```

Comment:
Just an observation: For embedded 2d grids (dfn), the new boundary tag treatment is not fully developed?

Reply:
I've been doing this part only for 2d not embedded.